packages/apex-lsp-testbed/package.json

```diff
@@ -33,8 +33,48 @@
     "start:jorje:debug:suspend": "node --enable-source-maps --inspect ./out/cli.js --server jorje --interactive --workspace https://github.com/trailheadapps/dreamhouse-lwc.git --suspend",
     "start:jorje:debug:verbose:suspend": "node --enable-source-maps --inspect ./out/cli.js --server jorje --interactive --verbose --workspace https://github.com/trailheadapps/dreamhouse-lwc.git --suspend",
     "start:apex-ls": "node --enable-source-maps ./out/cli.js --server nodeServer --interactive --workspace https://github.com/trailheadapps/dreamhouse-lwc.git",
-    "lint": "eslint src test",
-    "lint:fix": "eslint src test --fix"
+    "lint": "wireit",
+    "lint:fix": "wireit"
+  },
+  "wireit": {
+    "compile": {
+      "command": "tsc --build",
+      "dependencies": ["../apex-parser-ast:precompile"],
+      "output": ["out/**"],
+      "files": ["src/**/*.ts", "tsconfig*.json"]
+    },
+    "test": {
+      "command": "jest --json --outputFile=.wireit/test-results.json || true",
+      "dependencies": ["compile"],
+      "output": [".wireit/test-results.json"],
+      "files": ["src/**/*.ts", "test/**/*.ts", "jest.config.cjs"]
+    },
+    "test:quality": {
+      "command": "jest --testMatch \"**/test/**/*.quality.ts\"",
+      "dependencies": ["compile"],
+      "files": ["src/**/*.ts", "test/**/*.quality.ts", "jest.config.cjs"]
+    },
+    "test:coverage": {
+      "command": "jest --coverage --coverageDirectory=./coverage",
+      "dependencies": ["compile"],
+      "output": ["coverage/**"],
+      "files": ["src/**/*.ts", "test/**/*.ts", "jest.config.cjs"]
```

Comment:
for all these `test` scripts, I see them point to the jest config of the package in `files` but they should also point to the repo-level jest config that those inherit from.

Or you might be able to `**/jest.config.cjs` to get them all?